### PR TITLE
Bootstack page update

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Updated the BootStack page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/cloud/managed-cloud](http://0.0.0.0:8001/cloud/managed-cloud)
- Check that the copy matches the copy doc

## Details
Copy doc: https://docs.google.com/document/d/1g38PPPhEP0Z66eM_gezfUyUwAr5e9vGfTHJDWV8gZlE/edit

### CODE REVIEW

File: templates/cloud/managed-cloud.html [hunk 1]
```diff
@@ -5,158 +5,247 @@
...
-        </div>
-        <div class="row">
-          <div class="col-7">
-            <p>BootStack is a managed service where the world-class team at Canonical build and operate your OpenStack cloud either on-premises or hosted.</p>
-            <p>We provide the in-house expertise to stand-up and scale your cloud. And, when you&rsquo;re ready, we will hand over operational control.</p>
-            <p><a href="/cloud/contact-us?product=cloud-managed-cloud" class="p-button--brand">Schedule a demo</a></p>
-          </div>
-        </div>
+        <h1 class="p-heading--two">Fully managed OpenStack, by Canonical</h1>
+        <p class="p-heading--four">Build, Operate and Transfer Ubuntu OpenStack</p>
```
Comment (deadlight) [inline templates/cloud/managed-cloud.html]: Sentence case this

File: static/sass/_settings_colors.scss [hunk 2]
```diff
@@ -1,4 +1,4 @@
 $color-brand:             #e95420;
 $color-brand-light:       lighten($color-brand, 10%);
 $color-brand-dark:        darken($color-brand, 10%);
-$color-link:              $color-brand;
```
Comment (deadlight) [inline static/sass/_settings_colors.scss]: Don't remove this yet!

File: templates/cloud/managed-cloud.html (hunk 1 above)
Comment (anthonydillon) [inline templates/cloud/managed-cloud.html]: Good spot.

File: static/sass/_settings_colors.scss (hunk 2 above)
Comment (anthonydillon) [inline static/sass/_settings_colors.scss]: Done.

**Comment by frankiegranato** [comment]
Before I can give this a +1, Peter has made some changes that need to be incorporated - they are in the copydoc but I’m afraid that Peter hasn’t highlighted them so you’ll have to look at the history. 
The changes are mainly to the heading (it shorter) and the resources row (it has moved up).  But he says that he there are some other small amends in the copydoc but he hasn’t highlighted them so you’ll have to look at the history. 
Also, Ant in the future would you mind sending me an direct email or a ping when you need me to review a demo? I'm so sorry to ask, I know I'm supposed to be looking for git notifications myself - but ... what can I say - I'm a bad person :-(

**Comment by joanasa89** [comment]
Thanks for this.
Please see attached the design. Few changes:
-intro - new colour + illustration
- Resources - using the new pattern
- "Canonical secures..." section - revert the order - copy and illustration (reduce by 30%)
- "Fits your hybrid..." section - copy changes + use cards for the bullet point section(s)
- "Your data centre" + "...meets our Openstack" sections - use background colour #f7f7f7
- "Shape it to your needs" section - links should be orange + use the grid pattern (same as "your data centre"

cloud-managed-cloud-page
Dropbox folder: https://www.dropbox.com/sh/wovoqrekcvkt2kr/AAA5jJQyo8z_oZusysTXf1v9a?dl=0

**Comment by anthonydillon** [comment]
Updated the copy and the design to match.

**Comment by deadlight** [comment]
@yaili can you pick up this one, please?

@frankiegranato this is ready for re-review.

**Comment by yaili** [comment]
Design review:
- [ ] Resources doesn't match the design
- [ ] In "Canonical secures the OpenStack majority", remove the col classes from paragraphs, and place the text block in a col-7 instead of col-9
- [ ] In "Canonical secures the OpenStack", remove the top and bottom margin from figure
- [ ] In "Canonical secures the OpenStack", add prefix-1 to the image col
- [ ] In "Canonical secures the OpenStack", the paragraph after the figure shouldn't have any margin bottom, is this using the new vertical spacing?
- [ ] In "Fits your hybrid cloud strategy", the heading that starts "The right location for a workload" should be a paragraph with full stop at the end. I've confirmed this with @pmahnke

**Comment by anthonydillon** [comment]
@yaili thanks, ready for another review.

**Comment by yaili** [comment]
@anthonydillon I don't see any changes

**Comment by deadlight** [comment]
@yaili I updated the demo (see bot comment, above)

**Comment by yaili** [comment]
@deadlight has the pattern in Resources been implemented anywhere else before? The design has the icons smaller and the text larger and it seems to be grey. I understand that the demo has implemented just patterns, but it looks a bit odd. @spencerbygraves @kwm14 any thoughts?

**Comment by deadlight** [comment]
@yaili yes, it looks like this on MAAS: http://www.ubuntu.com-pr-2106.run.demo.haus/server/maas

**Comment by deadlight** [comment]
@yaili yeah, as @anthonydillon's now away for a while, I'll pick this up to get it landed.

**Comment by anthonydillon** [comment]
I knew the icons were larger then the design but I didn't want to create a
whole new pattern when we have one already.

Now that this is required in two places it should be proposed as a modifier
in Vanilla.

**Comment by deadlight** [comment]
@yaili I've reused the style that we had on MAAS:

http://www.ubuntu.com-pr-2106.run.demo.haus/cloud/managed-cloud

Compare:
http://www.ubuntu.com-pr-2106.run.demo.haus/server/maas

**Pull request merged**